loader.rb

```diff
@@ -128,6 +128,12 @@ def clover_freeze
   # this Ruby standard library method patches core classes.
   "".unicode_normalize(:nfc)
 
+  # Aws SDK started to autoload modules when used, so we need to load them
+  # before freezing. https://github.com/aws/aws-sdk-ruby/pull/3105
+  Aws::S3.const_get(:Client)
+  Aws::S3.const_get(:Presigner)
+  Aws::S3.const_get(:Errors)
```

Comment:
<img width="734" alt="Screenshot 2024-10-09 at 22 28 49" src="https://github.com/user-attachments/assets/78ad4c69-9647-4991-8148-993c6be4150a">

Rubocop fails with the following error: `Lint/Void: Constant Aws::S3::Client used in void context.`

What do you think about something like that?
```ruby
[:Client, :Presigner, :Errors].each { Aws::S3.const_get(_1) }
```

Reply:
Sure I guess, I suppose it is admirably short, which of course sells it, though I guess it would be fine to also use a pragma comment to turn that off that linter constraint; this is a particular part of the code where we, in fact, do want to do weird stuff for side effects and do not want to be bothered about "void context"